yt/yt/ytlib/query_tracker_client/proto/query_tracker_service.proto

```diff
@@ -0,0 +1,147 @@
+package NYT.NQueryTrackerClient.NProto;
```

Comment:
Is it better to make a dependency instead of duplication, like this?
```
message TReqStartQuery
{
    required NYT.NApi.NRpcProxy.NProto.TReqStartQuery rpcProxyRequest = 1;
    required string additionalField1 = 2;
    ...
}
```
It will save us from writing mapping between structures

Reply:
This dependency will not save us from mapping between structures. To do this, you will have to remove common code for different proxies in the native client.

Also, this dependency will remove the ability for us to change the protocol only in the RPC API or only between qt <-> proxies. But maybe it's not so necessary..

Reply:
Сhanged it, as you suggested